app/views/shoppinglist/index.html.erb

```diff
@@ -0,0 +1,27 @@
+
+<div class="recipes-container">
+<h1>Shopping List</h1>
+
+  <h2 class="recipes-shopping">
+    Amount of food items to buy: <%= @total_foods %>
+  </h2>
+
+  <h2 class="recipes-shopping">
+    Total value of food needed: $<%= @total_price %>
+  </h2>
```

Comment:
- [ ] @tea great work on your list however the amount of items to buy and total value of food needed should be on the same line to match the wireframe design as seen below 
Wireframe Design
![image](https://user-images.githubusercontent.com/61408860/222665343-d3b7a84f-9b5f-4c30-a41d-4a26f6168539.png)

Your Design
![image](https://user-images.githubusercontent.com/61408860/222665770-67f3c8ca-1acb-40b7-a7c6-d5d85c07c5b8.png)